code/modules/crafting/recipes/recipes_weapon_and_ammo.dm

```diff
@@ -119,7 +119,7 @@
 	tools = list(TOOL_WELDER)
 	time = 45
 	category = CAT_WEAPONRY
-	subcategory = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
```

Comment:
this is why i kept saying cat_weapon

Reply:
Don't mind me im just a huge Grunga-brain.